contrib/endpoints/src/api_manager/request_handler.cc

```diff
@@ -79,6 +105,35 @@ void RequestHandler::AttemptIntermediateReport() {
 // Sends a report.
 void RequestHandler::Report(std::unique_ptr<Response> response,
                             std::function<void(void)> continuation) {
+  if (api_manager_->IsConfigLoadingInProgress()) {
+    if (pending_request_callback_exist_) {
```

Comment:
no need for this check,  we can register pending for both check and report.  remove this flag